### PR TITLE
Coming from pandas

### PR DESCRIPTION
**Pull request by braaannigan**
I've added some big picture differences between Polars and Pandas. I've also fleshed out some of the examples that are there a bit more.

Let me know if you think it's the right direction

### CODE REVIEW

File: user_guide/src/coming_from_pandas.md [hunk 1]
```diff
@@ -1,44 +1,155 @@
...
 
 Let's go through some typical `Pandas` code and see how we might write that in `Polars`.
 
-## Column assignment
+### Selecting data
+
+As there is no index in `Polars` there is no `.loc` or `iloc` method in `Polars` - and
+there is also no `SettingWithCopyWarning` in `Polars`.
+
+To learn more about how you select data in `Polars` see the [indexing](indexing.md)
```
Comment (ritchie46) [inline user_guide/src/coming_from_pandas.md:68]: I would want to add here that in idiomatic polars use, accessing data by index is much less prevalent as we use the expression API to access our data.

File: user_guide/src/coming_from_pandas.md [hunk 2]
```diff
@@ -1,44 +1,155 @@
...
+### `Polars` uses Apache Arrow arrays to represent data in memory while `Pandas` uses `Numpy` arrays
+
+`Polars` represents data in memory with Arrow arrays while `Pandas` represents data in
+memory in `Numpy` arrays. Apache Arrow is an emerging standard for in-memory columnar
+analytics that can accelerate data load times, reduce memory usage and accelerate
+calculations.
+
+`Polars` can convert data to `Numpy` format with the `to_numpy` method.
+
+### `Polars` has more support for parallel operations than in `Pandas`
```
Comment (ritchie46) [inline user_guide/src/coming_from_pandas.md:29]: ~than in pandas~ -> than pandas

File: user_guide/src/coming_from_pandas.md [hunk 3]
```diff
@@ -1,44 +1,155 @@
...
+The CSV file has numerous columns but we just want to do a groupby on one of the id
+columns (`id1`) and then sum by a value column (`v1`). In `Pandas` this would be:
+
+```python
+    df = pd.read_csv(csvFile)
+    groupedDf = df.loc[:,['id1','v1']].groupby('id1').sum('v1')
+```
 
-### `Pandas`
+In `Polars` uou can build this query in lazy mode with query optimization and evaluate
```
Comment (ritchie46) [inline user_guide/src/coming_from_pandas.md:88]: ~uou~ -> you

**Review by ritchie46 — COMMENTED** [review]
Thanks, I've left a few comments.

Could you remove the other files from this PR, so we only merge the `coming from pandas` file?

**Comment by ritchie46** [comment]
Thanks @braaannigan. Sorry for the confusion about pre-commit.

In any case, I want to merge this, but there are still changes on files that should not be here I believe. There are 11 changed files and most don't have any change other than formatting.

Can you only add the following files?

* CONTRIBUTING.md
* requirements.txt
* user_guide/src/coming_from_pandas.md

**Comment by braaannigan** [comment]
@ritchie46 Just the relevant files now

**Comment by ritchie46** [comment]
> @ritchie46 Just the relevant files now

Thanks! Going in!

**Pull request merged**